Tasks/DownloadPackageV0/task.loc.json

```diff
@@ -9,8 +9,8 @@
   "author": "ms-vscs-rm",
   "version": {
     "Major": 0,
-    "Minor": 202,
-    "Patch": 1
+    "Minor": 210,
```

Comment:
I don't see task.json here. Is that already updated?

Reply:
yea it is already updated to minor 210.